sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -1694,6 +1747,40 @@ public partial class CassandraTableResourceInfo
         public Azure.ResourceManager.CosmosDB.Models.CassandraSchema Schema { get { throw null; } set { } }
         public string TableName { get { throw null; } set { } }
     }
+    public partial class ClientEncryptionIncludedPath
+    {
+        public ClientEncryptionIncludedPath(string path, string clientEncryptionKeyId, string encryptionType, string encryptionAlgorithm) { }
+        public string ClientEncryptionKeyId { get { throw null; } set { } }
+        public string EncryptionAlgorithm { get { throw null; } set { } }
+        public string EncryptionType { get { throw null; } set { } }
+        public string Path { get { throw null; } set { } }
+    }
+    public partial class ClientEncryptionKeyGetPropertiesResource : Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource
+    {
+        public ClientEncryptionKeyGetPropertiesResource() { }
+        public Azure.ETag? ETag { get { throw null; } }
+        public string Rid { get { throw null; } }
+        public float? Timestamp { get { throw null; } }
+    }
+    public partial class ClientEncryptionKeyGetResultCreateOrUpdateContent
+    {
+        public ClientEncryptionKeyGetResultCreateOrUpdateContent(Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource resource) { }
+        public Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource Resource { get { throw null; } }
```

Comment:
Shall we rename this property to `Properties`?

Reply:
@kr-santosh please take a look at this.

Reply:
I think this should be fine. Its similar to CosmosDBSqlContainerCreateOrUpdateContent

Reply:
# resolved